components/ChipTextField/examples/MDCChipTextFieldScrollView.m

```diff
@@ -0,0 +1,223 @@
+                                   relatedBy:NSLayoutRelationEqual
+                                      toItem:self
+                                   attribute:NSLayoutAttributeTop
+                                  multiplier:1
+                                    constant:0];
+  NSLayoutConstraint *contentConstraintBottom =
+      [NSLayoutConstraint constraintWithItem:contentView
+                                   attribute:NSLayoutAttributeBottom
+                                   relatedBy:NSLayoutRelationEqual
+                                      toItem:self
+                                   attribute:NSLayoutAttributeBottom
+                                  multiplier:1
+                                    constant:0];
+  NSLayoutConstraint *contentConstraintLeading =
+      [NSLayoutConstraint constraintWithItem:contentView
+                                   attribute:NSLayoutAttributeLeading
+                                   relatedBy:NSLayoutRelationEqual
+                                      toItem:self
+                                   attribute:NSLayoutAttributeLeading
+                                  multiplier:1
+                                    constant:0];
+  NSLayoutConstraint *contentConstraintTrailing =
+      [NSLayoutConstraint constraintWithItem:contentView
+                                   attribute:NSLayoutAttributeTrailing
+                                   relatedBy:NSLayoutRelationEqual
+                                      toItem:self
+                                   attribute:NSLayoutAttributeTrailing
+                                  multiplier:1
+                                    constant:0];
+  contentConstraintTrailing.priority = UILayoutPriorityDefaultLow;
+  NSLayoutConstraint *contentConstraintHeight =
+      [NSLayoutConstraint constraintWithItem:contentView
+                                   attribute:NSLayoutAttributeHeight
+                                   relatedBy:NSLayoutRelationEqual
+                                      toItem:self
+                                   attribute:NSLayoutAttributeHeight
+                                  multiplier:1
+                                    constant:0];
+  NSLayoutConstraint *contentConstraintWidth =
+      [NSLayoutConstraint constraintWithItem:contentView
+                                   attribute:NSLayoutAttributeWidth
+                                   relatedBy:NSLayoutRelationEqual
+                                      toItem:self
+                                   attribute:NSLayoutAttributeWidth
+                                  multiplier:1
+                                    constant:0];
+  contentConstraintWidth.priority = UILayoutPriorityDefaultLow;
+  [NSLayoutConstraint activateConstraints:@[
+    contentConstraintTop, contentConstraintBottom, contentConstraintLeading,
+    contentConstraintTrailing, contentConstraintHeight, contentConstraintWidth
+  ]];
```

Comment:
Why activating the constraints? I also don't see where these are being added to the view hierarchy.
Perhaps adding a comment explaining the constraint logic?

Reply:
Activating the constraints automatically add the constraints to the correct views.